network-api/networkapi/buyersguide/templates/catalog.html

```diff
@@ -39,11 +39,14 @@ <h1 class="h1-heading text-center mt-0">{% trans "Be Smart. Shop Safe." %}</h1>
       <label for="product-filter-search-input" class="clear-icon">&nbsp;</label>
     </span>
 
-    <span class="current-creepiness">
-      <span class="speech-bubble d-none">
-        <span class="text"></span>
-        <span class="hook"></span>
-      </span>
+    <span class="creep-o-meter-information">
+      <p class="speech-bubble-container">
+        <span class="speech-bubble">
+          <img class="hook" src="{% static "_images/buyers-guide/icon-bubble-hook.svg" %}">
+          <span class="text">{% trans "Scroll to see how creepy<br>people find these products!" %}</span>
```

Comment:
I tried to keep things CSS background based, but it just become way too complicated. An `<img>` specifically for just the speech bubble "hook" drastically simplified things ^^

Reply:
there's no variables that need to be localized (which is usually what we use blocktrans for) so I don't think so, but let's confirm with @TheoChevalier

Reply:
Confirming `trans` works here (but I had to check, I wasn’t 100% sure!)